gym_ergojr/envs/ergo_reacher_env.py

```diff
@@ -18,7 +18,7 @@
 class ErgoReacherEnv(gym.Env):
 
   def __init__(self,
-               headless=False,
+               headless=True,
```

Comment:
Same thing as above - what's the reasoning?

Reply:
Oh yeah! I am doing this because I am having some issues while running the experiments on the cloud. Even if `headless=True` it is still displaying the GUI. This is because in ErgoReacherAugmentedEnv class it calls the super().__init__() first. In that case `headless=False` and hence we can see the GUI. So, I changed headless to True. I know that this is not a great way to do this. But just to run the experiments on the cloud I did this.

Reply:
Ah-haaaa! That's the problem. 
In `ergo_reacher_augmented.py`, in Line 33, https://github.com/fgolemo/gym-ergojr/blob/ba357a9e475bff8ded298b4329baace9102c165a/gym_ergojr/envs/ergo_reacher_augmented.py#L33

...you are calling the init function on the parent without any parameters. We should give it the same parameters!

Reply:
That could also explain some performance discrepancies between identical environments with an augmented/non-augmented version because this way it's using the default parameters for the parent.

Reply:
> this way it's using the default parameters for the parent
No, because we are calling `ErgoReacherAugmentedEnv.__init__(all the arguments)` in the very next line. So, it's not using the default parameters (I printed out and checked it).

Reply:
Anyways I fixed everything now